qcodes/dataset/dependencies.py

```diff
@@ -0,0 +1,48 @@
+from typing import Dict, Any
+
+from qcodes.dataset.param_spec import ParamSpec
+
+
+class InterDependencies:
+    """
+    Object containing the ParamSpecs of a given run
+    """
+
+    def __init__(self, *paramspecs: ParamSpec) -> None:
+        self.paramspecs = paramspecs
+
+    def __repr__(self) -> str:
+        output = self.__class__.__name__
+        output += '('
```

Comment:
Could this simply be 

output +="(" + " ".join(self.paramspecs) + ")"

Reply:
With an additional comma, yes. Fixed.